addons/mail/controllers/discuss.py

```diff
@@ -303,6 +306,23 @@ def mail_message_remove_reaction(self, message_id, content):
     # Channel API
     # --------------------------------------------------------------------------
 
+    @http.route('/mail/channel/add_guest_as_member', methods=['POST'], type='json', auth='public')
+    def mail_channel_add_guest_as_member(self, channel_id, channel_uuid, **kwargs):
+        channel_sudo = request.env['mail.channel'].browse(channel_id).sudo().exists()
```

Comment:
This is why I left the `int()` even when it wasn't necessary. Because otherwise people fail to copy paste when it is actually necessary.

I would suggest to keep the `int()` always in all the routes.